### ISSUE TITLE
Contrastive loss details

### ISSUE THREAD

**Issue by bernaljg** [kandorm/CLINE#4]
Hi! I found something a bit confusing that you might be able to clarify. For the contrastive loss implemented in "LecbertForPreTraining" class, you use a sigmoid as the activation function before applying the BCE loss but in the paper you mention a normal InfoNCE loss (with softmax activations) was used. Am I misunderstanding something perhaps?

**Comment by HarBatt** [comment]
Yeah, the author used a simple BCE loss to maximize similarity. Weird, it's different from the paper.
